src/widgets/auto-style/category.js

```diff
@@ -14,23 +14,61 @@ module.exports = AutoStyler.extend({
     return StyleUtils.replaceWrongSpaceChar(style);
   },
 
+  getRange: function () {
+    var model = this.dataviewModel;
+    var categories = model.get('data');
+    var range = [];
```

Comment:
Not wrong, just a matter of taste but why not writing this in a more functional style:
```
return _.map(categories, function (category) {
    return this.colors.getColorByCategory(category.name);
}, this)
```